src/test/java/se/curity/examples/spark/integration/AbstractApiAuthorizationTest.java

```diff
@@ -23,67 +32,52 @@
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 import java.util.Map;
+import java.util.UUID;
 import java.util.logging.Logger;
 
 import static com.github.tomakehurst.wiremock.client.WireMock.get;
 import static com.github.tomakehurst.wiremock.client.WireMock.ok;
-import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig;
 
 public abstract class AbstractApiAuthorizationTest {
 
-    static final String ISSUER = "jwtIssuer";
-    static final String AUDIENCE = "someClientId";
+    static final String ISSUER = "http://localhost:8443/oauth/v2/oauth-anonymous";
+    static final String AUDIENCE = "api.example.com";
+    static final String JWKS_PATH = "/oauth/v2/oauth-anonymous/jwks";
     static final String SCOPE = "products";
     static final int PORT = 9090;
 
-    static MockProductServiceImpl mockProductService;
-    static boolean applicationStarted;
+    private static boolean started = false;
 
     /**
-     * Creates JWTs for the given issuer and key
+     * Creates JWTs for the given issuer and a generated key ID
      */
-    static MockJwtIssuer mockJwtIssuer = new MockJwtIssuer(ISSUER, "key1");
+    static MockJwtIssuer mockJwtIssuer = new MockJwtIssuer(ISSUER, UUID.randomUUID().toString());
     /**
      * Used to mock JWKS endpoint for mocked JWT issuer
      */
-    @RegisterExtension
-    static WireMockExtension wm1 = WireMockExtension.newInstance()
-            .options(wireMockConfig().httpDisabled(true).httpsPort(8443))
-            .build();
+    static WireMockServer mockAuthorizationServer;
 
     @BeforeAll
-    static synchronized void startApplication() throws ServletException {
-        String jwksUrl = wm1.baseUrl() + "/jwks";
-        Logger.getLogger(AbstractApiAuthorizationTest.class.getName()).info("Mocked JWKS URL on " + jwksUrl);
+    public static void startMockAuthorizationServer() throws ServletException {
 
-        ServerOptions options = new ServerOptions();
-        options.setPort(PORT);
-        options.setJwksUrl(jwksUrl);
-        options.setIssuer(ISSUER);
-        options.setAudience(AUDIENCE);
-        options.setScope(SCOPE);
-
-        if (!applicationStarted) {
-            mockProductService = new MockProductServiceImpl();;
-            SparkServerExample.runLocally(mockProductService, options);
-            applicationStarted = true;
+        if (started) {
+            return;
         }
-    }
 
-    @BeforeEach
-    void publishJwks() {
-        wm1.stubFor(get("/jwks")
+        started = true;
+        var options = new WireMockConfiguration().port(8443).httpDisabled(false);
```

Comment:
no need to specify `httpDisabled(false)`?

Reply:
removed - better